locales/cy/translations.json

```diff
@@ -276,6 +276,11 @@
             "limitedPartnerUsualResidentialAddress": {
                 "title": "WELSH - Where is the limited partner's usual residential address?",
                 "publicRegisterInformationLine1": "WELSH - We will not show the limited partner's usual residential address on the public register."
+            },
+            "limitedPartnerPrincipalOfficeAddress": {
+                "title": "WELSH -Where is the limited partner's principal office address?",
```

Comment:
"WELSH -" is inconsistent with the others, needs an extra space after it.

Reply:
Updated